README.md

```diff
@@ -3,3 +3,45 @@
 Wizualizacja wyznaczania optymalnego kąta trasy łodzi. TODO
 
 Autorzy: Bartłomiej Szlachta, Kamil Rusin, Kornel Fox
+
+##Proste Przeszukiwanie Lokalne
+
+###Wprowadzenie
+Jest to proces iteracyjny, który przemieszcza się w przestrzeni od początkowego rozwiązania
+do kolejnego według ustalonej reguły. Generuje on nowe rozwiązania randomistycznie, dokładniej
+ mówiąc agenci w sąsiedztwie są generowane losowo według współczynnika sąsiedztwa. Każdy 
+ start algorytmu po takiej samej liczbie kroków otrzymuje inny rezultat.
+ 
+###Algorytm
+W naszym problemie algorytm został zaimplementowany następująco. 
+Znajduje on najlepszy kąt, pod którym łódka przebędzie najmniejszą odległość do punktu
+docelowego. Pod uwagę brane są współczynniki takie jak: 
+- początkowy kąt łódki względem celu,
+- prędkość łódki,
+- prędkość nurtu rzeki,
+- szerokość rzeki,
+- odległość do punktu docelowego w linii prostej,
```

Comment:
Ta sekcja nie dotyczy de facto algorytmu, a ogólnie rozwiązywanego problemu. Gdy będę opisywał problem, przeniosę to do odpowiedniej sekcji. Wtedy możliwe, że coś trzeba będzie jeszcze porobić w tej sekcji dot. algorytmu, zastanowimy się później.